scenarios1/07_The_Return.cfg

```diff
@@ -91,10 +91,18 @@
             speaker=Efraim_de_Ceise
             message= _ "Wait, do not leave... I will miss you."
         [/message]
+#ifndef HARD
         [message]
             speaker=Delenia
             message= _ "Finally you ceased to show off before your new subject of desire. I will leave soon, so if I am carrying some things you want back, take them back."
         [/message]
+#else
+        [message]
+            speaker=Delenia
+            message= _ "Finally you ceased to show off before your new subject of desire."
```

Comment:
Two mistakes there.  She should have said she would be leaving.  And the hint about stripping gear probably should be included the first time, in case it's the first time through for the player.  Although the later is a bit unclear, since units do leave without warning as well.  But my intention was you get a warning the first time, but after that (such as when Argan leaves the first time) you're expected to know what to do by now.

Reply:
Aha, okay.